packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,140 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[
+            $style['c-mediaElement'],
+            (stacked ? $style['c-mediaElement--stack'] : ''),
+            (reverse ? $style['c-mediaElement--reverse'] : '')
+        ]">
+        <div
+            :class="[
+                $style['c-mediaElement-content'],
+                contentAlignClass
+            ]">
+            <h3
+                :class="[
+                    $style['c-mediaElement-title'],
+                    fontSizeClass
```

Comment:
Just a preference, as it would mean only adding one class instead of two, but you could put the size class on the `mediaElement` element as a modifier and then just have the title and text react to that outer class.

```
<div class="c-mediaElement c-mediaElement--fontSizeLarge">
    …
    <h3 class="c-mediaElement-title">
    </h3>
</div>
```

```scss
.c-mediaElement--fontSizeLarge & {
    & .c-mediaElement--title {
        font-size: xxx;
    }
    & .c-mediaElement--text {
        …
    }
}
```

Reply:
I like this idea, makes sense. WIll make the change

Reply:
I have gone along with this suggestion, however I have now added a `$font-sizes` map. Using this map I loop over to create the appropriate sizes, reason for this is I think it was starting to bloat out the css with repeated parts, which could be solved with a loop. @ashleynolan could you please take a look again and see if this suits you.

Reply:
👍 Think that looks good